client/api_payment.go

```diff
@@ -0,0 +1,84 @@
+package client
+
+import (
+	"context"
+	"cosmossdk.io/math"
+	gnfdSdkTypes "github.com/bnb-chain/greenfield/sdk/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	paymentTypes "github.com/bnb-chain/greenfield/x/payment/types"
+)
+
+type Payment interface {
+	GetStreamRecord(ctx context.Context, streamAddress string) (*paymentTypes.StreamRecord, error)
+
+	Deposit(ctx context.Context, toAddress string, amount math.Int, txOption *gnfdSdkTypes.TxOption) (string, error)
+	Withdraw(ctx context.Context, fromAddress string, amount math.Int, txOption *gnfdSdkTypes.TxOption) (string, error)
+	DisableRefund(ctx context.Context, paymentAddress string, txOption *gnfdSdkTypes.TxOption) (string, error)
```

Comment:
As I said here. https://github.com/bnb-chain/greenfield-go-sdk/pull/48#discussion_r1170872321

Suggest not use pointer for gnfdSdkTypes.TxOption for unified interface style.

Reply:
fixed